src/containers/DAO/pages/DAOList/DAOTable/DAOTableCard.tsx

```diff
@@ -41,7 +41,7 @@ const DAOTableCard: React.FC<DAOTableCardProps> = ({ data }) => {
 		if (totalUsd) {
 			add({ zna: zna, total: totalUsd });
 		}
-	}, [totalUsd]);
+	}, [add, data.zna, totalUsd]);
```

Comment:
these missing dependents were causing the stat widget total to be incorrect on DAO homepage

Reply:
Nice find! Can't wait to deprecate this 🥲